.goreleaser.yml

```diff
@@ -0,0 +1,39 @@
+before:
+  hooks:
+    - go mod tidy
```

Comment:
i think this is unnecessary, instead add -mod=readonly somewhere in the builds section (later we can set up a github action for validating this before merging).

please also add to the top of README.md:

```
## Project status

At this point the project is primerly intended to be used by checking it out and modifying main.go, however you can also use it by downloading the binaries from releases.
```

and add a CONTRIBUTING.md, with

```
Contributions are welcome in the form of merge requests/issues. Please alway run `go mod tidy` before submitting a merge request.
```